.github/workflows/security-code-scanner.yml

```diff
@@ -5,39 +5,34 @@ on:
     branches:
       - main
   pull_request:
-    branches:
-      - main
   workflow_dispatch:
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
-  # This cancels in-progress workflows for all branches except main as it is, we can remove the rest of the conditions as the first one acts as break circuit with the OR
   cancel-in-progress: ${{ !(contains(github.ref, 'refs/heads/main') || contains(github.ref, 'refs/heads/stable')) }}
 
 jobs:
   run-security-scan:
-    runs-on: ubuntu-latest
+    uses: MetaMask/action-security-code-scanner/.github/workflows/security-scan.yml@v2
     permissions:
       actions: read
       contents: read
       security-events: write
-    steps:
-      - name: MetaMask Security Code Scanner
-        uses: MetaMask/Security-Code-Scanner@main
-        with:
-          repo: ${{ github.repository }}
-          paths_ignored: |
-            test/
-            docs/
-            storybook/
-            .storybook/
-            '**/*.test.js'
-            '**/*.test.ts'
-            '**/*.test.jsx'
-            '**/*.test.tsx'
-            '**/*.stories.tsx'
-            development/chromereload.js
-            node_modules
-          rules_excluded: example
-          project_metrics_token: ${{secrets.SECURITY_SCAN_METRICS_TOKEN}}
-          slack_webhook: ${{ secrets.APPSEC_BOT_SLACK_WEBHOOK }}
+    with:
+      repo: ${{ github.repository }}
+      scanner-ref: v2
```

Comment:
Is this needed or only for testing?

Reply:
reusable workflows by default use default branch, this is required if we want to specify ref it should be using, v2 release in this case